content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -192,6 +192,7 @@ metadata:
   name: foo-pvc
   namespace: foo
 spec:
+  storageClassName: ""
```

Comment:
Do we want to also make and empty storageClassName in the PV, below? I know it's the default but it might nudge people into noticing that they have to match.

Reply:
Done and added comment on pvc